### ISSUE TITLE
[SF][ObjectPage]: [a11y]Heading levels should only increase by one, and Heading hierarchy is not correctly defined on the page

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents-react#7043]
### Describe the bug

**Incorrect Heading Levels in Different Scenarios**
When **Tabstrip** Is Present for Multiple Sections Navigation:

1. The first section tab is missing a heading level.
2. If a form is configured inside a subsection, its title is assigned an H6 heading, skipping the H5 heading.

When Tabstrip Is **NOT** Present:

1. The subsection title is assigned an H4 heading, skipping the H3 heading level.
2. If a form is configured inside a subsection, its title is assigned an H6 heading, skipping the H5 heading.

### Reproduction steps

1.Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-objectpage--default
2. Turn on screen reader
3. Focus to Object status
4. Check the announcement

### Expected Behaviour

Heading levels should only increase by one
Ensure the order of headings is semantically correct.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/09bb75bc-61a9-4009-b0c9-a507cca969f1)

### UI5 Web Components for React Version

2.7.2

### UI5 Web Components Version

2.7.3

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

Successfactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

> The first section tab is missing a heading level.

This will be fixed with the linked PR.

> If a form is configured inside a subsection, its title is assigned an H6 heading, skipping the H5 heading.

There's currently no API to change the heading level of `FormGroup`s. Please open a [feature request](https://github.com/SAP/ui5-webcomponents/issues/new/choose) for this in the ui5-webcomponents repo.

> When Tabstrip Is NOT Present:

This is not supported. An `ObjectPage` should always implement `ObjectPageSection`s (except when a placeholder is rendered), if this is not required the `DynamicPage` can be used.
